### PR TITLE
Ruby on Rails capstone project - Budget app

### PR DESCRIPTION
**Pull request by Mhdez221993**
## Project requirements
[Project presentation](https://www.loom.com/share/ef99e5295aaa4e1f9b74bcda42334ea6)

* You should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
* Splash screen
* Sign up and log in pages
* Home page (catagories page)
* Transactions page
* "Add a new category" page
* "Add a new transaction" page
* Create unit and integration tests for all the most important components of your RoR application

splash

login_payeasy

sigup_payeasy

cetegory_payeasy

transaction_payeasy

## Thank you for review my project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,51 @@
-# README
+# Pay Easy
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+The Ruby on Rails capstone project (remember what they are?) is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-Things you may want to cover:
+[live](https://obscure-spire-53238.herokuapp.com/)
```
Comment (emmanuellekamwa) [inline README.md:6]: - [x] I have seen that you have not added the link to your video presentation. Please add it to your readme file.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: CHANGES REQUESTED 🛑❌**
Hello @Mhdez221993 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Mhdez221993) [reply README.md:6]: Hi I just added

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @Mhdez221993 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**